src/nuget.config

```diff
@@ -1,5 +1,9 @@
 <configuration>
   <config>
-    <add key="repositoryPath" value="../lib" />
+    <add key="repositoryPath" value="../packages" />
```

Comment:
This is something I need getting used to. If we want to go with packages, maybe we should not include the ../ either?